test/kitchen/test/integration/system-probe-test/rspec_datadog/system-probe-test_spec.rb

```diff
@@ -28,6 +28,7 @@
 TIMEOUTS = {
   "pkg/network/protocols/http$" => "15m",
   "pkg/network/tracer$" => "25m",
+  "pkg/network/usm$" => "15m",
```

Comment:
```suggestion
TIMEOUTS = {
  "pkg/network/usm$" => "15m",
  "pkg/network/tracer$" => "25m",
```

Reply:
it is not about sorting, `pkg/network/protocols/http` does not exist anymore

Reply:
That directory still exists on my branch?